primitives/src/lib.rs

```diff
@@ -195,5 +249,59 @@ pub mod v0 {
 				assert_eq!(error, DecodingFailed,);
 			});
 		}
+
+		// Compare all the different `DispatchError` variants with the expected `Error`.
+		#[test]
+		#[cfg(feature = "runtime")]
```

Comment:
This should not be required if the "std" feature is added.

Reply:
Without `runtime` feature enabled, the conversion `Error::from` does not work because `From<DispatchError>` is only enabled when there is `runtime`.